dev-debug/bpftrace/bpftrace-0.21.2-r1.ebuild

```diff
@@ -92,4 +93,6 @@ src_configure() {
 src_install() {
 	cmake_src_install
 	doman man/man8/*.?
+	gunzip "${WORKDIR}/man/man8/bpftrace.8.gz"
```

Comment:
`|| die`, but I can add it while merging after testing this.

Reply:
Thanks! Please do.

Reply:
thanks all, I've pushed